src/ucp/wireup/signaling_ep.c

```diff
@@ -1,208 +0,0 @@
-
-    packed_size = uct_ep_tag_eager_bcopy(proxy_ep->uct_ep, tag, 0,
-                                         ucp_signaling_ep_pack_tag_short, &ctx,
-                                         UCT_SEND_FLAG_SIGNALED);
-    if (packed_size < 0) {
-        return (ucs_status_t)packed_size;
-    }
-
-    ucp_proxy_ep_replace(proxy_ep);
-    return UCS_OK;
-}
-
-static ssize_t
-ucp_signaling_ep_tag_eager_bcopy(uct_ep_h ep, uct_tag_t tag, uint64_t imm,
-                                 uct_pack_callback_t pack_cb, void *arg,
-                                 unsigned flags)
-{
-    ucp_proxy_ep_t *proxy_ep = ucs_derived_of(ep, ucp_proxy_ep_t);
-    ssize_t packed_size;
-
-    packed_size = uct_ep_tag_eager_bcopy(proxy_ep->uct_ep, tag, imm, pack_cb,
-                                         arg, flags | UCT_SEND_FLAG_SIGNALED);
-    if (packed_size >= 0) {
-        ucp_proxy_ep_replace(proxy_ep);
-    }
-    return packed_size;
-}
-
-static ucs_status_t
-ucp_signaling_ep_tag_eager_zcopy(uct_ep_h ep, uct_tag_t tag, uint64_t imm,
-                                 const uct_iov_t *iov, size_t iovcnt,
-                                 unsigned flags, uct_completion_t *comp)
-{
-    ucp_proxy_ep_t *proxy_ep = ucs_derived_of(ep, ucp_proxy_ep_t);
-    ucs_status_t status;
-
-    status = uct_ep_tag_eager_zcopy(proxy_ep->uct_ep, tag, imm, iov, iovcnt,
-                                    flags | UCT_SEND_FLAG_SIGNALED, comp);
-
-    if (!UCS_STATUS_IS_ERR(status)) {
-        ucp_proxy_ep_replace(proxy_ep);
-    }
-    return status;
-}
-
-static ucs_status_ptr_t
-ucp_signaling_ep_tag_rndv_zcopy(uct_ep_h ep, uct_tag_t tag, const void *header,
-                                unsigned header_length, const uct_iov_t *iov,
-                                size_t iovcnt, unsigned flags,
-                                uct_completion_t *comp)
-{
-    ucp_proxy_ep_t *proxy_ep = ucs_derived_of(ep, ucp_proxy_ep_t);
-    ucs_status_ptr_t status;
-
-    status = uct_ep_tag_rndv_zcopy(proxy_ep->uct_ep, tag, header, header_length,
-                                   iov, iovcnt, flags | UCT_SEND_FLAG_SIGNALED,
-                                   comp);
-    if (!UCS_PTR_IS_ERR(status)) {
-        ucp_proxy_ep_replace(proxy_ep);
-    }
-    return status;
-}
-
-static ucs_status_t
-ucp_signaling_ep_tag_rndv_request(uct_ep_h ep, uct_tag_t tag,
-                                  const void* header, unsigned header_length,
-                                  unsigned flags)
-{
-    ucp_proxy_ep_t *proxy_ep = ucs_derived_of(ep, ucp_proxy_ep_t);
-    ucs_status_t status;
-
-    status = uct_ep_tag_rndv_request(proxy_ep->uct_ep, tag, header,
-                                     header_length,
-                                     flags | UCT_SEND_FLAG_SIGNALED);
-    if (status == UCS_OK) {
-        ucp_proxy_ep_replace(proxy_ep);
-    }
-    return status;
-}
-
-ucs_status_t ucp_signaling_ep_create(ucp_ep_h ucp_ep, uct_ep_h uct_ep,
```

Comment:
remove the function declaration from `wireup.h` file

Reply:
fixed